CHANGELOG/CHANGELOG-1.5.md

```diff
@@ -1,5 +1,19 @@
 # Changelog 1.5
 
+## 1.5.1 - 2022-03-07
+
+### Changed
+
+- Use iptables-wrapper in Antrea container. Now antrea-agent can work with distros that lack the iptables kernel module of "legacy" mode (ip_tables). ([#3308](https://github.com/antrea-io/antrea/pull/3308), [@antoninbas])
+- Reduce permissions of Antrea ServiceAccount for updating annotations. ([#3408](https://github.com/antrea-io/antrea/pull/3408), [@tnqn])
+
+### Fixed
+
+- Fix NodePort/LoadBalancer Service cannot be accessed when externalTrafficPolicy changed from Cluster to Local with proxyAll enabled. ([#3380](https://github.com/antrea-io/antrea/pull/3380), [@hongliangl])
+- Fix initial egress connections from Pods may go out with node IP rather than Egress IP. ([#3378](https://github.com/antrea-io/antrea/pull/3378), [@tnqn])
+- Fix NodePort Service access when the Service and an Egress selects the same Pod. ([#3397](https://github.com/antrea-io/antrea/pull/3397), [@hongliangl])
```

Comment:
selects -> select

Or "when an Egress selects the same Pod as the NodePort Service".

Reply:
updated. thanks!